BE/OwnershipAndControl/Executives.rdf

```diff
@@ -506,6 +540,14 @@
 		<skos:definition>indicates the party that endorses, enables, empowers, or gives permission in the situation</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-be-oac-exec;hasCorporateOfficer">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;isAffectedBy"/>
+		<rdfs:label>has corporate officer</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-be-oac-cpty;ControlledParty"/>
```

Comment:
No. I don't think one can say that an officer "controls" an org in the formal sense used in FIBO. More like "manages" or "operates".

Reply:
From a regulatory and finance perspective, every corporate officer has some level of control over the organization they manage. Officers have the legally delegated authority to act on behalf of the organization, including entering into contracts, acting as a signatory, etc.

Reply:
Not really according to &fibo-fnd-oac-ctl;Control which has definition: "situation in which some party possesses something (explicitly or implicitly), or has the power to direct or strongly influence the direction of the management and policies related to it, whether through the ownership of voting shares, by contract, or otherwise"